addons/mail/static/src/core/web/message_model_patch.js

```diff
@@ -0,0 +1,102 @@
+import { Message } from "@mail/core/common/message_model";
+
+import {
+    deserializeDate,
+    deserializeDateTime,
+    formatDate,
+    formatDateTime,
+} from "@web/core/l10n/dates";
+import { _t } from "@web/core/l10n/translation";
+import { formatFloat } from "@web/core/utils/numbers";
+import { patch } from "@web/core/utils/patch";
+import {
+    formatChar,
+    formatInteger,
+    formatMonetary,
+    formatText,
+} from "@web/views/fields/formatters";
+
+patch(Message.prototype, {
+    /**
+     * @returns {string}
+     */
+    formatTracking(trackingType, trackingValue) {
```

Comment:
`formatTracking` and `formatTrackingOrNone` don't make use of `this`, there is no need to have them on message model.

They can probably be moved to utils format instead?